mmcv/cnn/bricks/hsigmoid.py

```diff
@@ -8,19 +8,23 @@
 class HSigmoid(nn.Module):
     """Hard Sigmoid Module. Apply the hard sigmoid function:
     Hsigmoid(x) = min(max((x + bias) / divisor, min_value), max_value)
-    Default: Hsigmoid(x) = min(max((x + 1) / 2, 0), 1)
+    Default: Hsigmoid(x) = min(max((x + 3) / 6, 0), 1)
+
+    Note:
+        In MMCV v1.4.3, we modified the default value of args to align with
+        PyTorch official.
 
     Args:
-        bias (float): Bias of the input feature map. Default: 1.0.
-        divisor (float): Divisor of the input feature map. Default: 2.0.
+        bias (float): Bias of the input feature map. Default: 3.0.
+        divisor (float): Divisor of the input feature map. Default: 6.0.
         min_value (float): Lower bound value. Default: 0.0.
         max_value (float): Upper bound value. Default: 1.0.
 
     Returns:
         Tensor: The output tensor.
     """
 
-    def __init__(self, bias=1.0, divisor=2.0, min_value=0.0, max_value=1.0):
+    def __init__(self, bias=3.0, divisor=6.0, min_value=0.0, max_value=1.0):
```

Comment:
We need to print warning information that the default value has been changed.

Reply:
Done.